src/routes/home.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+
+router.get('/', async (req, res) => {
+  try {
+    res.sendFile(path.resolve('../src/static/home.html'));
+  } catch (e) {
+    res.status(404).render('error-page', { title: "404 Page Not Found", error: true });
```

Comment:
Change to 500 status because it would be our server failing if it errors.

Reply:
Ok changed it